client/src/components/questionsAnswers/D-QuestionsCards.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import AnswersCards from './E-AnswersCards.jsx';
+import SubModals from './M-SubmissionModals.jsx';
+
+const { useState, useEffect } = React;
+
+function QuestionsCards(props) {
+  const { question, productName } = props;
+  const [answers, setAnswers] = useState([]);
+  const [openModal, setOpenModal] = useState(false);
+
+  const { question_body, question_id, question_helpfulness } = question;
+
+  useEffect(() => {
+    axios.get(`/qa/questions/${question_id}/answers`)
+      .then((response) => setAnswers(response.data.results));
+  }, []);
+
+  // TODO: HANDLE PHOTOS IN AXIOS POST REQUEST
+  function handleSubmit(text, nickname, userEmail) {
```

Comment:
More of a suggestion rather than a critical/immediate request, I recommend when naming functions to choose a word with meaning that provides some context